### ISSUE TITLE
Tags  and time not displayed in dark mode under RUM video page

### ISSUE THREAD

**Issue by neha00290** [openobserve/openobserve#1723]
### Which OpenObserve functionalities are the source of the bug?

Don't known / other

### Is this a regression?

Yes

### Description

Steps
- Switch to dark mode
-  Go to any video under RUM
- Select tags tab beside breadcrums option.
- Observe
<img width="1424" alt="Screenshot 2023-10-23 at 8 34 42 PM" src="https://github.com/openobserve/openobserve/assets/144308844/30bb4cb7-120a-45c6-b52a-69f2eb94aa42">

### Please provide the version you discovered this bug in (check about page for version information)

```true
Version
Commit Hash
Build Date
v0.6.4
be240a29e2679eb0e279f2a3ef645fdae48b75d7
2023-10-23T10:56:48Z
```

**Comment by omkarK06** [comment]
This issue has been fixed in #1745

**Comment by neha00290** [comment]
testing complete. closing this ticket as it works fine
